hassio/addon-view/hassio-addon-view.html

```diff
@@ -23,7 +23,7 @@
       }      
       .content {
         padding: 24px 0 32px;
-        max-width: 600px;
+        max-width: 800px;
```

Comment:
I don't think that this is a good idea. We build all our UIs mobile first. By keeping it at 600px, people will not go crazy in their READMEs

Reply:
GitHub has max-width:980px

Reply:
all in all we should move the long_description to a dialog. you can do it like the changelog